File: Feitos em C/2779.c
#include <stdio.h>
int main() {
	int pictures,buy,cont=0,i,j;
	scanf("%d %d",&pictures,&buy);
	int vet[buy],memo[buy];
	for(i=0;i<buy;i++){
		scanf("%d",&vet[i]);
		memo[i]=vet[i];
	}
	for(i=0;i<buy;i++){
		for(j=i+1;j<buy;j++){
			if(vet[i]==memo[j]&&memo[j]!=-1){
				memo[j]=-1;
			}
		}
	}
	for(i=0;i<buy;i++){
		if(memo[i]!=-1){
			cont++;
		}
	}
	printf("%d\n",pictures-cont);
    return 0;
}

File: Feitos em C Sharp/2779.cs
using System;
namespace ConsoleApp6 {
    class Program {
        static void Main(string[] args) {
			int pictures = int.Parse(Console.ReadLine());
			int cont = 0;
			int buy = int.Parse(Console.ReadLine());
			int[] vet = new int[buy];
			int[] memo = new int[buy];
			for (int i = 0; i < buy; i++) {
				vet[i] = int.Parse(Console.ReadLine());
				memo[i] = vet[i];
			}
			for (int i = 0; i < buy; i++) {
				for (int j = i + 1; j < buy; j++) {
					if (vet[i] == memo[j] && memo[j] != -1) {
						memo[j] = -1;
					}
				}
			}
			for (int i = 0; i < buy; i++) {
				if (memo[i] != -1) {
					cont++;
				}
			}
			Console.WriteLine(pictures - cont);
		}
    }
}
